### DISCUSSION TITLE
How do I select the last character on a line?

### DISCUSSION THREAD

**Discussion by knpwrs** [ggandor/lightspeed.nvim#113] [Q&A] [answered]
I have the following code:

```ts
<FastField id={token} key={token} name={token} |onChange={() => console.log('fastfieldchange')}>
```
My cursor is where the `|` is. I would like to delete to the final `>`, so I type `dz>` and get the following highlights:

![image](https://user-images.githubusercontent.com/174864/146846718-75c82639-7c7e-4db0-9632-e65eb56534e9.png)

It looks like I should press `s`, but there's another `s` highlighted before the final `s`. What should I press to delete to the final `>`?

**Comment by ggandor** [comment, marked as answer]
> It looks like I should press s, but there's another s highlighted before the final s

The first `s` is a shortcut, the second is not, it's a normal label (note the different highlight), so you should enter the second character of the match, even though the label is already visible. You might find [this video](https://youtu.be/ESyld9NCl1w) helpful, if the usage is not 100% clear from the readme.

To answer the specific question: `{char}<enter>` to the rescue (https://github.com/ggandor/lightspeed.nvim#matching-line-breaks-linewise-motions). The `¬` character before the label indicates that you should press enter as the second char to reach that match.

**Reply by knpwrs** [reply]
I could have sworn I tried that! Well, that appears to work. Thank you!
